File: src/test/java/com/znsio/teswiz/screen/calculator/CalculatorScreen.java
package com.znsio.teswiz.screen.calculator;


public abstract class CalculatorScreen {

    public static CalculatorScreen get() {
        return com.znsio.teswiz.screen.ScreenRegistry.getScreen(CalculatorScreen.class);
    }

    public abstract CalculatorScreen handlePopupIfPresent();

    public abstract CalculatorScreen selectNumber(String number);

    public abstract CalculatorScreen pressOperation(String operation);
}

File: src/test/resources/playwright/screens/calculator/calculator.screen.ts
import { unsupportedScreenAction } from "../screen-route.ts";

export function handlePopupIfPresent(): object {
  return unsupportedScreenAction(
    "Calculator is not supported on web for WEB_ENGINE=playwright-ts.",
  );
}

export function selectNumber(): object {
  return unsupportedScreenAction(
    "Calculator is not supported on web for WEB_ENGINE=playwright-ts.",
  );
}

export function pressOperation(): object {
  return unsupportedScreenAction(
    "Calculator is not supported on web for WEB_ENGINE=playwright-ts.",
  );
}
